test/e2e/packagemanifest_e2e_test.go

```diff
@@ -2,170 +2,196 @@ package e2e
+				Expect(err).ToNot(HaveOccurred())
+				defer cleanupCatalogSource()
+
+				_, err = fetchCatalogSource(crc, catalogSourceName, testNamespace, catalogSourceRegistryPodSynced)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("querying the package manifest using the package name", func() {
+
+					expectedStatus := packagev1.PackageManifestStatus{
+						CatalogSource:          catalogSourceName,
+						CatalogSourceNamespace: testNamespace,
+						PackageName:            packageName,
+						Channels: []packagev1.PackageChannel{
+							{
+								Name:           stableChannel,
+								CurrentCSV:     packageStable,
+								CurrentCSVDesc: packagev1.CreateCSVDescription(&csv, string(csvJSON)),
+							},
+						},
+						DefaultChannel: stableChannel,
+					}
+
+					pm, err := fetchPackageManifest(pmc, testNamespace, packageName, packageManifestHasStatus)
+					Expect(err).ToNot(HaveOccurred(), "error getting package manifest")
+					Expect(pm).ShouldNot(BeNil())
+
+					Expect(pm.GetName()).Should(Equal(packageName))
+					Expect(pm.Status).Should(Equal(expectedStatus))
+					Expect(pm.Status.Channels[0].CurrentCSVDesc.MinKubeVersion).Should(Equal("0.0.0"))
+					Expect(pm.Status.Channels[0].CurrentCSVDesc.RelatedImages[0]).Should(Equal(*dummyImage))
+					Expect(pm.Status.Channels[0].CurrentCSVDesc.NativeAPIs).Should(Equal(csv.Spec.NativeAPIs))
+					Expect(pm.GetLabels()["projected"]).Should(Equal("label"))
+					Expect(pm.GetLabels()["operatorframework.io/arch.amd64"]).Should(Equal("supported"))
+					Expect(pm.GetLabels()["operatorframework.io/os.linux"]).Should(Equal("supported"))
+
+					// Get a PackageManifestList and ensure it has the correct items
+					pmList, err := pmc.OperatorsV1().PackageManifests(testNamespace).List(metav1.ListOptions{})
+					Expect(err).NotTo(HaveOccurred(), "could not access package manifests list meta")
+					Expect(pmList.ListMeta).ShouldNot(BeNil(), "package manifest list metadata empty")
+					Expect(pmList.Items).ShouldNot(BeNil())
+				})
+
+			})
 		})
-		require.NoError(GinkgoT(), err, "package-server not available")
-
-		_, err = fetchCatalogSource(GinkgoT(), crc, source.GetName(), testNamespace, catalogSourceRegistryPodSynced)
-		require.NoError(GinkgoT(), err)
-
-		pm, err := fetchPackageManifest(GinkgoT(), pmc, testNamespace, packageName, packageManifestHasStatus)
-		require.NoError(GinkgoT(), err, "error getting package manifest")
-		require.NotNil(GinkgoT(), pm)
-		require.Equal(GinkgoT(), packageName, pm.GetName())
-
-		relatedImages := pm.Status.Channels[0].CurrentCSVDesc.RelatedImages
-		require.Equal(GinkgoT(), len(expectedRelatedImages), len(relatedImages))
+	})
 
-		for _, v := range relatedImages {
-			_, ok := expectedRelatedImages[v]
-			require.True(GinkgoT(), ok, "Expect this image %s to exist in the related images list\n", v)
-		}
+	Context("Given an operator-registry image", func() {
```

Comment:
Not a 100% sure, if the `Context` wording is accurate. I felt a second look at it would be good.

Reply:
Until we update this test to build its own registry image during the test run, it makes more sense to me to merge the 
"CatalogSource created..." context into this and leave the image reference string as an implementation detail.